### PR TITLE
[Merged by Bors] - feat(AlgebraicGeometry) Add more API

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean [hunk 1]
```diff
@@ -409,52 +409,50 @@ lemma toOpen_comp_locallyRingedSpaceAdjunction_homEquiv_app
...
 
 theorem adjunction_homEquiv_symm_apply {X : Scheme} {R : CommRingCatᵒᵖ}
     (f : X ⟶ Scheme.Spec.obj R) :
-    (ΓSpec.adjunction.homEquiv X R).symm f = (locallyRingedSpaceAdjunction.homEquiv X.1 R).symm f :=
-  by rw [adjunction_homEquiv]; rfl
+    (ΓSpec.adjunction.homEquiv X R).symm f =
+      (locallyRingedSpaceAdjunction.homEquiv X.1 R).symm f := rfl
 #align algebraic_geometry.Γ_Spec.adjunction_hom_equiv_symm_apply AlgebraicGeometry.ΓSpec.adjunction_homEquiv_symm_apply
 
 set_option backward.isDefEq.lazyWhnfCore false in -- See https://github.com/leanprover-community/mathlib4/issues/12534
```
Comment (semorrison) [inline Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean:437]:
```suggestion
```
Can be removed now.

Comment (semorrison) [reply Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean:437]: (There's another in this file that can also be removed. Happy if these get done separately.)

File: Mathlib/AlgebraicGeometry/Scheme.lean [hunk 2]
```diff
@@ -215,52 +285,43 @@ theorem inv_val_c_app {X Y : Scheme} (f : X ⟶ Y) [IsIso f] (U : Opens X) :
...
+theorem Spec_toLocallyRingedSpace (R : CommRingCat) :
     (specObj R).toLocallyRingedSpace = Spec.locallyRingedSpaceObj R :=
   rfl
-#align algebraic_geometry.Scheme.Spec_obj_to_LocallyRingedSpace AlgebraicGeometry.Scheme.specObj_toLocallyRingedSpace
+#align algebraic_geometry.Scheme.Spec_obj_to_LocallyRingedSpace AlgebraicGeometry.Spec_toLocallyRingedSpace
 
 /-- The induced map of a ring homomorphism on the ring spectra, as a morphism of schemes.
 -/
-def specMap {R S : CommRingCat} (f : R ⟶ S) : specObj S ⟶ specObj R :=
+def SpecMap {R S : CommRingCat} (f : R ⟶ S) : specObj S ⟶ specObj R :=
```
Comment (semorrison) [inline Mathlib/AlgebraicGeometry/Scheme.lean:304]: This seems dubious.

Comment (erdOne) [reply Mathlib/AlgebraicGeometry/Scheme.lean:304]: Do you mean misleading?
I believe the naming convention allow names that are capitalized in the literature as exceptions?
Or else `Scheme.Spec` should be `Scheme.spec` according to the convention.

Comment (erdOne) [reply Mathlib/AlgebraicGeometry/Scheme.lean]: I've removed the renaming from this PR.

File: Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean (hunk 1 above)
Comment (erdOne) [reply Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean]: Nice to know, thanks!

**Comment by mattrobball** [comment]
!bench

File: Mathlib/AlgebraicGeometry/Morphisms/ClosedImmersion.lean [hunk 3]
```diff
@@ -91,7 +91,7 @@ theorem spec_of_surjective {R S : CommRingCat} (f : R ⟶ S) (h : Function.Surje
 /-- For any ideal `I` in a commutative ring `R`, the quotient map `specObj R ⟶ specObj (R ⧸ I)`
 is a closed immersion. -/
 instance spec_of_quotient_mk {R : CommRingCat.{u}} (I : Ideal R) :
-    IsClosedImmersion (Scheme.specMap (CommRingCat.ofHom (Ideal.Quotient.mk I))) :=
+    IsClosedImmersion (Scheme.Spec.map (CommRingCat.ofHom (Ideal.Quotient.mk I)).op) :=
```
Comment (alreadydone) [inline Mathlib/AlgebraicGeometry/Morphisms/ClosedImmersion.lean:94]: What's going on here?

**Comment by semorrison** [comment]
bors d+

File: Mathlib/AlgebraicGeometry/Morphisms/ClosedImmersion.lean (hunk 3 above)
Comment (erdOne) [reply Mathlib/AlgebraicGeometry/Morphisms/ClosedImmersion.lean:94]: Just that `specMap` is now in the AlgebraicGeometry namespace so I chose a random fix.

**Comment by erdOne** [comment]
Thanks!
maintainer merge

**Comment by erdOne** [comment]
Oops wrong command
bors merge

**Pull request closed without merging**